package.json

```diff
@@ -20,10 +20,16 @@
     "nodemon": "^1.18.7"
   },
   "dependencies": {
-    "dotenv": "^8.0.0",
+    "bootstrap": "^3.4.1",
```

Comment:
Check if these are required on the server side. Believe it should be on the client side (React)

Reply:
removed bootstrap and moved bootstrap-social to client side